integration-tests/api/helpers/order-seeder.js

```diff
@@ -183,4 +184,97 @@ module.exports = async (connection, data = {}) => {
     price: 1000,
     data: {},
   });
+
+  const orderWithClaimAndSwapAndFulfillmentAndReturn = manager.create(Order, {
```

Comment:
Here adding an order which contains fulfillments, swaps, claims, returns ... for the needed integration tests